recipe_recommender/bodybuilding.py

```diff
@@ -9,6 +9,12 @@
 import requests
 import json
 
+DATA_DIR = '../data/'
+CLEANED_DATA_DIR = '../data/cleaned/'
```

Comment:
Use `os.path.join` here for compatibility with systems (like windows) where the forward slash is used in file paths.

Reply:
I also moved these to the "global" space in my function, but used a leading `_` to indicate they are "private" variables. I don't typically use all caps in variable names.  We should decide on a convention for this.

Reply:
What version of python are you using? Some people I work with have been talking about new path stuff in python 3 and you don't use os but it's still compatible. maybe we should check it out.

Reply:
I am using python 3.6. I haven't heard anything about that, but it's possible

Reply:
https://realpython.com/python-pathlib/

?